support-models/src/main/scala/com/gu/support/workers/User.scala

```diff
@@ -11,13 +11,13 @@ case class User(
   firstName: String,
   lastName: String,
   billingAddress: Address,
-  deliveryAddress: Option[Address] = None,
+  deliveryAddress: Option[Address] = None,// TODO should later be moved out of User and into the relevant case class
   telephoneNumber: Option[String] = None,
   allowMembershipMail: Boolean = false,
   allowThirdPartyMail: Boolean = false,
   allowGURelatedMail: Boolean = false,
   isTestUser: Boolean = false,
-  deliveryInstructions: Option[String] = None
+  deliveryInstructions: Option[String] = None// TODO should later be moved out of User and into the relevant case class
```

Comment:
this is home delivery paper specific so shouldn't be provided everywhere, but that's not for this PR.  Delivery address above is specific to physical products only.